src/fiskaltrust.ifPOS/v1/de/Models/TseInfo.cs

```diff
@@ -6,37 +6,119 @@ namespace fiskaltrust.ifPOS.v1.de
     [DataContract]
     public class TseInfo
     {
+        /// <summary>
+        /// maximal number of clientId's supported
+        /// </summary>
         [DataMember(Order = 10)]
         public long MaxNumberOfClients { get; set; }
 
+        /// <summary>
+        /// current number of clients registered
+        /// </summary>
         [DataMember(Order = 20)]
         public long CurrentNumberOfClients { get; set; }
 
+        /// <summary>
+        /// current list of clientId's registered
+        /// </summary>
         [DataMember(Order = 30)]
         public IEnumerable<string> CurrentClientIds { get; set; }
 
+        /// <summary>
+        /// maximal number of started transactions supported
+        /// </summary>
         [DataMember(Order = 40)]
         public long MaxNumberOfTransactions { get; set; }
 
+        /// <summary>
+        /// current number of started transactions
+        /// </summary>
         [DataMember(Order = 50)]
         public long CurrentNumberOfTransactions { get; set; }
 
-        [DataMember(Order = 60)]
-        public string TsePublicKeyBase64 { get; set; }
+        /// <summary>
+        /// current list of started stransactions
+        /// </summary>
+        [DataMember(Order =60)]
+        public IEnumerable<ulong> CurrentTransactionNumberss { get; set; }
 
-        [DataMember(Order = 70)]
-        public string TseSerialNumberOctet { get; set; }
+        /// <summary>
+        /// maximum number of signatures
```

Comment:
I think this comment is not really helpful, because it says the exact same thing as the property name - and the same applies for many in this file. Can we be more precise? Here, we could e.g. write "Maximum number of signatures that can be processed by this TSE before it expires."

If we just repeat the name of the property in the summary, we can IMO skip adding those comments.